collect_app/src/test/java/org/odk/collect/android/formlists/blankformlist/BlankFormListItemTest.kt

```diff
@@ -0,0 +1,162 @@
+package org.odk.collect.android.formlists.blankformlist
+
+import android.net.Uri
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.odk.collect.forms.Form
+import org.odk.collect.forms.instances.Instance
+import org.odk.collect.formstest.InMemInstancesRepository
+import org.odk.collect.projects.Project
+
+@RunWith(AndroidJUnit4::class)
+class BlankFormListItemTest {
+    private val instancesRepository = InMemInstancesRepository()
+
+    @Test
+    fun `Form should be properly converted to BlankFormListItem`() {
```

Comment:
It looks like you do enough work in the other tests in this file to just have a single form and instance here to check a `Form` to `BlankFormListItem` conversion where everything is populated. I might have missed an edge case this is capturing, but if so I think that should have its own test.

Reply:
Right after adding more tests in this class this one seems to be to big. Fixed.